G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice.Tests/Commands/OutlookClose/ShouldCloseOutlook_WhenCloseCommandIsCalled.G1ANT

```diff
@@ -0,0 +1,7 @@
+﻿outlook.open display false
```

Comment:
Why do you use here parameter `display` false, but you don't to that in some other places? More consistency :-)

Reply:
I only do it with outlook, because word and excel are not supporting it.